Content.Shared/Pulling/Systems/SharedPullingSystem.Actions.cs

```diff
@@ -59,7 +59,7 @@ public bool CanPull(EntityUid puller, EntityUid pulled)
                 return false;
             }
 
-            if (!_containerSystem.IsInSameOrNoContainer(puller, pulled))
+            if(_containerSystem.IsEntityInContainer(puller))
```

Comment:
Still need to check if target in container

Reply:
there is a ballmer peak of sleep deprivation and I have missed that by miles